### ISSUE TITLE
Integration using entry_type

### ISSUE THREAD

**Issue by Sjorsa** [golles/ha-knmi#16]
### Did you read the instructions?

- [X] I have read the [README](../blob/main/README.md), including the [known-limitations](../blob/main/README.md#known-limitations) section.

### The problem

There is an error in the logs that says an integration is using entry_type and that it will be deprecated in 2022.3.
I looked through all my custom integrations and this one uses `entry_type`.

### What version of this integration has the issue?

newest

### What version of Home Assistant Core has the issue?

2021.12.9

### Home Assistant log

```shell
2022-01-12 16:56:34 WARNING (MainThread) [homeassistant.helpers.frame] Detected code that uses str for device registry entry_type. This is deprecated and will stop working in Home Assistant 2022.3, it should be updated to use DeviceEntryType instead. Please report this issue.
```

**Comment by golles** [comment]
Which version of this integration are you using?
I think this has already been resolved in 1.1.3 https://github.com/golles/ha-knmi/commit/40ceb29869961f3c12453610012245b0cf270268#diff-b3440da3bd2c347e45d7dd8bf02f9b398ebb589ed0416b781f5991c5f1d37d8b

**Comment by Sjorsa** [comment]
I think it mightve been another integration. Sorry for the hassle!
